tests/supervisor/test_execution.py

```diff
@@ -4,29 +4,35 @@
 import pytest
 from aleph_message.models import ItemHash
 
-from aleph.vm.conf import settings
+from aleph.vm.conf import Settings, settings
 from aleph.vm.controllers.firecracker import AlephFirecrackerProgram
 from aleph.vm.models import VmExecution
 from aleph.vm.orchestrator import metrics
+from aleph.vm.orchestrator.messages import load_updated_message
 from aleph.vm.storage import get_message
 
 
 @pytest.mark.asyncio
-async def test_create_execution():
+async def test_create_execution(mocker):
     """
     Create a new VM execution and check that it starts properly.
     """
+    mock_settings = Settings()
+    mocker.patch("aleph.vm.conf.settings", new=mock_settings)
+    mocker.patch("aleph.vm.storage.settings", new=mock_settings)
+    mocker.patch("aleph.vm.controllers.firecracker.executable.settings", new=mock_settings)
+    mocker.patch("aleph.vm.controllers.firecracker.program.settings", new=mock_settings)
```

Comment:
Would a loop help here ?

```suggestion
    for path in [
        "aleph.vm.conf.settings",
        "aleph.vm.storage.settings",
        "aleph.vm.controllers.firecracker.executable.settings",
        "aleph.vm.controllers.firecracker.program.settings"
    ]:
        mocker.patch(path, new=Settings())
```

Reply:
Frankly I find it less readable

Reply:
Yes, I see better to be more explicit here also.